### ISSUE TITLE
Bower install fails. Bump chartist dependency in bower.json

### ISSUE THREAD

**Issue by bridgeport** [Globegitter/chartist-plugin-tooltip#94]
Great plugin. In bower.json, you last bumped the chartist dependency as referenced here: https://github.com/Globegitter/chartist-plugin-tooltip/issues/26

Could you please bump this again, as I can't install this via bower as 1) I already have chartist installed and 2) your dependency is set to 0.94 and the latest version of chartist is 0.10.1. As far as I can tell, in my testing the latest version of this plugin works fine with 0.10.1.

Also, could you set the dependency to be a little more flexible so that minor versions of chartist can be upgraded without breaking the chartist-plugin-tooltip strict dependency requirement?

FYI, I'm getting the following:

```
$ bower install chartist-plugin-tooltip --save-dev
bower chartist-plugin-tooltip#* cached https://github.com/Globegitter/chartist-plugin-tooltip.git#0.0.17
bower chartist-plugin-tooltip#*         validate 0.0.17 against https://github.com/Globegitter/chartist-plugin-tooltip.git#*
bower chartist#~0.9.4                     cached https://github.com/gionkunz/chartist-js.git#0.9.8
bower chartist#~0.9.4                   validate 0.9.8 against https://github.com/gionkunz/chartist-js.git#~0.9.4
bower                                  ECONFLICT Unable to find suitable version for chartist
```

Thanks.

**Comment by fbonzon** [comment]
Correct. Current bower.json file of this plugin is not compatible with the latest Chartist release. Please increase required version.

Currently reads `~0.9.4` which means `>=0.9.4` and  `<0.10.0`, but latest Chartist is `0.10.1`.

**Comment by jamesjacobs** [comment]
Any update on this?

**Comment by dirak** [comment]
I have not gotten a chance to set this package up with bower yet. Would 3d857486989076f9eea6c5063c5898c329b4bc52 suffice? @bridgeport @jamesjacobs
